src/uu/pathchk/src/options.rs

```diff
@@ -0,0 +1,4 @@
+pub const POSIX: &str = "posix";
```

Comment:
please move that into pathchk.rs
it should not be in a separate file

Reply:
I thought it is a convention as 9081e12 keeps the options in a separate file. No issues, I will change it.

Reply:
Not sure what you mean. 
https://github.com/uutils/coreutils/blob/9081e120d6b866c94679d9e29c7bdb17ccccb4d0/src/uu/numfmt/src/numfmt.rs#L38-L47 
options is in the same file

Reply:
My bad 🤦 . My master is not up to date and I overlooked the commit. Thanks for clarifying. I will fix it.